content/en/platform/corda/4.12/enterprise/node/operating/tvu/tvu-overview.md

```diff
@@ -0,0 +1,27 @@
+---
+date: '2023-12-15'
+menu:
+  corda-enterprise-4-12:
+    identifier: corda-enterprise-4-12-tvu
+    parent: corda-enterprise-4-12-corda-nodes-operating
+tags:
+- tvu
+- transaction validator utility
+title: Transaction Validator Utility
+weight: 130
+---
+
+# Transaction Validator Utility
+
+Corda 4.12 has been released on and is compatible with JDK 17 and Kotlin 1.8. Therefore, migrating to Corda 4.12 means that you must update to a newer JDK and Kotlin version to align with the updated Corda dependencies.
+
+Before migrating to Corda 4.12, you must also ensure that the transactions committed to the database using the older Corda version are valid, verifiable and deserializable post-migration. You can do it using Transaction Validator Utility (TVU) which streams and performs validation on database transactions. TVU's useful features include:
```

Comment:
Note that the transaction validator utility is only useful for customers upgrading from a previous version of Corda. Would be worth mentioning this. Doesn't really say that here.
Customers using 4.12 fresh, with no previous version of Corda don't need to bother with transaction validator utilitity, as they will have no old transactions.